docs/api-types/backup.md

```diff
@@ -0,0 +1,130 @@
+# Backup API Type
+
+## Use
+
+The `Backup` API type is used as a request for the Ark Server to perform a backup. Once created, the
+Ark Server immediately starts the backup process.
+
+## API GroupVersion
+
+Backup belongs to the API group version `ark.heptio.com/v1`.
+
+## Definition
+
+Here is a sample `Backup` object with each of the fields documented:
+
+```yaml
+# Standard Kubernetes API Version declaration. Required.
+apiVersion: ark.heptio.com/v1
+# Standard Kubernetes Kind declaration. Required.
+kind: Backup
+# Standard Kubernetes metadata. Required.
+metadata:
+  # Backup name. May be any valid Kubernetes object name. Required.
+  name: a
+  # Backup namespace. Must be heptio-ark. Required.
+  namespace: heptio-ark
+# Parameters about the backup. Required.
+spec:
+  # Array of namespaces to include in the backup. If unspecified, all namespaces are included.
+  # Optional.
+  includedNamespaces:
+  - '*'
+  # Array of namespaces to exclude from the backup. Optional.
+  excludedNamespaces:
+  - some-namespace
+  # Array of resources to include in the backup. Resources may be shortcuts (e.g. 'po' for 'pods')
+  # or fully-qualified. If unspecified, all resources are included. Optional.
+  includedResources:
+  - '*'
+  # Array of resources to exclude from the backup. Resources may be shortcuts (e.g. 'po' for 'pods')
+  # or fully-qualified. Optional.
+  excludedResources:
+  - storageclasses.storage.k8s.io
+  # Whether or not to include cluster-scoped resources. Valid values are true, false, and
+  # null/unset. If true, all cluster-scoped resources are included (subject to included/excluded
+  # resources and the label selector). If false, no cluster-scoped resources are included. If unset,
+  # all cluster-scoped resources are included if and only if all namespaces are included and there are
+  # no excluded namespaces. Otherwise, if there is at least one namespace specified in either
+  # includedNamespaces or excludedNamespaces, then the only cluster-scoped resources that are backed
+  # up are those associated with namespace-scoped resources included in the backup. For example, if a
+  # PersistentVolumeClaim is included in the backup, its associated PersistentVolume (which is
+  # cluster-scoped) would also be backed up.
+  includeClusterResources: null
+  # Individual objects must match this label selector to be included in the backup. Optional.
+  labelSelector:
+    matchLabels:
+      app: ark
+      component: server
+  # Whether or not to snapshot volumes. This only applies to PersistentVolumes for Azure, GCE, and
+  # AWS. Valid values are true, false, and null/unset. If unset, Ark performs snapshots by default.
```

Comment:
for unset, maybe say "Ark performs snapshots if and only if a persistentVolumeProvider is configured"?

Reply:
Reworded, ptal